pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/RRDD.scala

```diff
@@ -16,12 +16,13 @@ private abstract class BaseRRDD[T: ClassTag, U: ClassTag](
     parent: RDD[T],
     numPartitions: Int,
     func: Array[Byte],
-    parentSerialized: Boolean,
+    parentSerialized: String,
```

Comment:
Let's rename these `parentSerialized` in the Scala classes to match the R names, e.g. `parentSerializedMode`.